src/Packetery/Core/Entity/Order.php

```diff
@@ -977,6 +991,19 @@ public function getNumber(): ?string {
 		return $this->number;
 	}
 
+	/**
+	 * Has order id.
+	 *
+	 * @return bool
+	 */
+	public function hasNumber(): bool {
```

Comment:
navrhuji použít ternární operátor

return null !== $this->number && '' !== $this->number;

Zde i u dalších has metod, které se přidávali v rámci tohoto PR